azkaban-web-server/src/web/js/azkaban/view/job-edit.js

```diff
@@ -260,4 +259,44 @@ $(function () {
   jobEditView = new azkaban.JobEditView({
     el: $('#job-edit-pane')
   });
+
+  formatJobPropertyValues();
+
 });
+
+
+// Improve readability of long job property values
+function formatJobPropertyValues() {
+  var patternsToFormat = new Map();
+
+  // Format values like:
+  // 1) --param1 value1 --param2 --param3 value3
+  // 2) -param1 value -param2 -param3
```

Comment:
Can you also add the properties after formatting in the comment (here and below). That way the before and after are both clear.

Reply:
Done.